admin/src/components/TextArrayInput/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { ChangeEffect, ToBeFixed } from '../../../../types';
+// @ts-ignore
+import { TextInput } from '@strapi/design-system/TextInput';
+
+interface IProps {
+  onChange: ChangeEffect<string[]>;
+  initialValue?: string[];
+  id?: string;
+  name?: string;
+  label?: string;
+  disabled?: boolean;
+  // TODO: [ @ltsNotMike ] Fix this before commit 
+  error?: ToBeFixed;
+}
+
+const TextArrayInput: React.FC<IProps> = ({ onChange, initialValue, ...props}) => {
+  const [value, setValue] = useState(!!initialValue ? initialValue.reduce((acc, cur) => acc+cur+"; ", "") : "");
```

Comment:
```suggestion
  const [value, setValue] = useState(Array.isArray(initialValue) ? initialValue.reduce((acc, cur) => [acc, cur, "; "].join(""), "") : "");
```

or alternatively you can use template string